### PR TITLE
The "three dots" context menu is not closed after unsuccessful licensing from Media Gallery

### LINKED ISSUE #1802: The "three dots" context menu is not closed after unsuccessful licensing from Media Gallery
### Preconditions (*)
1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Old Media Gallery disabled in Stores -> Configuration -> Advanced-> System -> Media Gallery -> Enable Old Media Gallery - No
4. Provided _API Key (Client ID)_ . **Not** provided _Client Secret_
client_key
5. Unlicensed image preview saved to Media Gallery being at the bottom of images

### Steps to reproduce (*)
1. Go to **Content - Media Gallery**, click **Search Adobe Stock**
2. Select the previously saved Unlicensed image
3. Click "three dots" and select **License**
4. Do not click anywhere, observe the behavior

### Expected result (*)
The context menu is closed after _Login failed..._ message appears 
contextmenu_expected

### Actual result (*)
The context menu is NOT closed after _Login failed..._ message appears
contextmenu
context

### PR DESCRIPTION
**Pull request by joweecaquicla**
### Description (*)

This PR will introduce the modification that will hide the context menu after unsuccessful licensing from Media Gallery.

### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1802

### Manual testing scenarios (*)

**Preconditions**

1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Old Media Gallery disabled in Stores -> Configuration -> Advanced-> System -> Media Gallery -> Enable Old Media Gallery -> No
4. Provide API Key (Client ID). Do not provide Client Secret.
5. Unlicensed image preview saved to Media Gallery being at the bottom of images

**Testing steps**

1. Go to Content - Media Gallery, click Search Adobe Stock
2. Select the previously saved Unlicensed image
3. Click "three dots" and select License
4. Do not click anywhere.
5. The context menu is closed after _Login failed_... message appears

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by joweecaquicla** [comment]
@magento run all

**Comment by rogyar** [comment]
Hi @joweecaquicla. Thank you for the fix. Frankly speaking, the solution looks a little bit "hacky". But at first glance, it looks like the image.js does not know about the collapsible object used in the template. 
Anyway, will you have a chance to cover this fix with an MFTF test? 

Thank you!

**Comment by joweecaquicla** [comment]
@magento run all tests

**Comment by joweecaquicla** [comment]
Hi @rogyar. The MFTF test for this fix will be on a separate PR to Adobe Stock Integration repo as the license action is only available when ASI modules are included.

File: app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js [hunk 1]
```diff
@@ -289,6 +291,13 @@ define([
             }
         },
 
+        /**
+         * Action to close the context menu in media gallery.
+         */
+        closeContextMenu: function () {
+            $(this.gridSelector).click();
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js:307]: I think this fix is risky and may interfere with other functionalities. I'll check if there are more localized approaches

**Comment by sivaschenko** [comment]
The MFTF test was created for adobe stock integration in https://github.com/magento/adobe-stock-integration/pull/1826
However it is currently skipped due to https://github.com/magento/adobe-stock-integration/issues/1170

**Comment by sivaschenko** [comment]
@magento run all tests

**Comment by sivaschenko** [comment]
@magento run Functional Tests EE,Functional Tests B2B

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-8199](https://jira.corp.magento.com/browse/ENGCOM-8199) has been created to process this Pull Request

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

The context menu is closed automatically 
contextmenu

**Pull request merged**